frontend/workflows/serverexperimentation/src/start-experiment.tsx

```diff
@@ -9,27 +9,16 @@ import { yupResolver } from "@hookform/resolvers/yup";
 import * as yup from "yup";
 
 import Dialog from "./dialog";
-import { FormContent } from "./form-content";
+import FormFields from "./form-fields";
 
 enum FaultType {
   ABORT = "Abort",
   LATENCY = "Latency",
 }
 
-const faultInjectionTypeItems = [
-  {
-    label: "Internal",
-    value: IClutch.chaos.serverexperimentation.v1.FaultInjectionCluster.FAULTINJECTIONCLUSTER_UPSTREAM.toString(),
-  },
-  {
-    label: "External (3rd party)",
-    value: IClutch.chaos.serverexperimentation.v1.FaultInjectionCluster.FAULTINJECTIONCLUSTER_DOWNSTREAM.toString(),
-  },
-];
-
 type ExperimentData = IClutch.chaos.serverexperimentation.v1.AbortFaultConfig &
   IClutch.chaos.serverexperimentation.v1.LatencyFaultConfig &
-  IClutch.chaos.serverexperimentation.v1.ClusterPairTarget & { type: FaultType };
+  IClutch.chaos.serverexperimentation.v1.ClusterPairTarget & { faultType: FaultType };
```

Comment:
change of the name of the property from `type` to `faultType` to make it clearer what is it for